lighty-core/lighty-controller/src/main/java/io/lighty/core/controller/api/LightyServices.java

```diff
@@ -77,9 +75,9 @@
 
     DistributedDataStoreInterface getOperationalDatastore();
 
-    DOMDataTreeShardingService getDOMDataTreeShardingService();
+    DistributedShardFactory getDOMDataTreeShardingService();
 
-    DOMDataTreeService getDOMDataTreeService();
+    DistributedShardFactory getDOMDataTreeService();
```

Comment:
these should be deprecated, shouldn't they?

and most probably referencing `getDistributedShardFactory` as replacement?

Reply:
done